Mathlib/Algebra/BigOperators/Field.lean

```diff
@@ -20,3 +21,33 @@ lemma Multiset.sum_map_div (s : Multiset ι) (f : ι → K) (a : K) :
 
 lemma Finset.sum_div (s : Finset ι) (f : ι → K) (a : K) :
     (∑ i ∈ s, f i) / a = ∑ i ∈ s, f i / a := by simp only [div_eq_mul_inv, sum_mul]
+
+-- TODO: Move these once `Finset.dens` doesn't depend on `Field` anymore
+namespace Finset
+variable {α β : Type*} [Fintype β]
+
+@[simp]
+lemma dens_disjiUnion (s : Finset α) (t : α → Finset β) (h) :
+    (s.disjiUnion t h).dens = ∑ a ∈ s, (t a).dens := by simp [dens, sum_div]
+
+variable {s : Finset α} {t : α → Finset β}
+
+lemma dens_biUnion [DecidableEq β] (h : ∀ x ∈ s, ∀ y ∈ s, x ≠ y → Disjoint (t x) (t y)) :
```

Comment:
Let's phrase this in terms of PairwiseDisjoint

Reply:
This matches `card_biUnion`

Reply:
That one should also be changed :)

Reply:
That's up to you, but note that my suggestion is also consistent with `sum_biUnion`